src/transformStateWithClones.js

```diff
@@ -8,6 +8,38 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const states = [];
+  const newState = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        for (const prop in action.extraData) {
+          newState[prop] = action.extraData[prop];
+        }
+
+        states.push({ ...newState });
```

Comment:
![image](https://user-images.githubusercontent.com/99403198/211158286-c60a24c7-dac6-4c3f-8c91-4f90ceb90031.png)
You repeat this action 3 times `states.push({ ...newState })`. Think about how to write it only once

Reply:
Thanks, fixed!